snapcraft/tests/test_plugin_tar_content.py

```diff
@@ -0,0 +1,83 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright (C) 2016 Canonical Ltd
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import os.path
+import tarfile
+from snapcraft.plugins.tar_content import TarContentPlugin
+from snapcraft.tests import TestCase
+
+
+class TestTarContentPlugin(TestCase):
+
+    def setUp(self):
+        super().setUp()
+        # setup the expected target dir in our tempdir
+        self.build_prefix = 'parts/tarContent/build/'
```

Comment:
just a nit, because it's not common to find camel case in python code, I would write this as tar_content.